nixos/modules/services/networking/unifi.nix

```diff
@@ -116,9 +116,13 @@ in
 
     users.users.unifi = {
       uid = config.ids.uids.unifi;
```

Comment:
you should switch unifi user and group to dynamically allocated user ids. See the discussion in #133166 for motivation.

Reply:
We can, but we will need a way for users to explicitly specify a user/group. Otherwise, the ownership of its data files would be messed up if `services.unifi.dataDir` is set to anywhere outside of its systemd `StateDirectory`.

But anyways, it's a separate thing and we shouldn't do it in this PR.

Reply:
This fixes the eval when using these modules, which got broken in #133166

Feel free to open a PR with a different fix for these modules @symphorien, but I'd just like for things to evaluate again, tbh.